virttest/ip_sniffing.py

```diff
@@ -291,9 +291,13 @@ def _output_handler(self, line):
 
         # BootP/DHCP (RFC 951/2131)
         if re.match(r"\d+\.\d+\.\d+\.\d+", packet[0]):
-            chaddr = packet[4]
-            yiaddr = packet[5]
-            m_type = packet[6]
+            try:
+                chaddr = packet[4]
+                yiaddr = packet[5]
+                m_type = packet[6]
+            except IndexError:
+                # Ignore problematical packets
+                return True
```

Comment:
@luckyh    why it return "True" here ?  Don't need to check anything for "problematical packet"  ?